rfm_examples/XBDP_SimpleTx.m

```diff
@@ -67,7 +67,7 @@
 sray.uri = uri;
 sray.Mode(:) = {'Tx'}; %set mode, 'Rx', 'Tx, 'Disabled'
 txPhaseCalOffsets = zeros(size(sray.TxGain));
-sray.TxAttn(:) = true; %1: Attenuation Off, 0: Attenuation On
+sray.TxAttn(:) = 1; %1: Attenuation Off, 0: Attenuation On
 sray.SteerTx(0,0,txPhaseCalOffsets); %Broadside
 sray.TxGain(:) = 127; %127: Highest Gain, 0: Lowest Gain, Decimal Value
 sray.LatchTxSettings; %Latch settings to devices
```

Comment:
What does this method do here?

Reply:
Added error() message to prevent Latch methods from getting called without establishing connection to hardware.